ui/components/component-library/button-icon/__snapshots__/button-icon.test.js.snap

```diff
@@ -0,0 +1,16 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ButtonIcon should render button element correctly 1`] = `
+<div>
+  <button
+    aria-label="add"
+    class="box mm-button-icon mm-button-icon--size-lg box--display-inline-flex box--flex-direction-row box--justify-content-center box--align-items-center box--color-icon-default box--background-color-transparent box--rounded-lg"
+    data-testid="button-icon"
+  >
+    <div
+      class="box icon icon--size-lg box--flex-direction-row box--color-inherit"
+      style="mask-image: url('./images/icons/icon-add-square-filled.svg;"
```

Comment:
Ah ok I see what you mean about the inline style for icon. It seems broken here.